rocAL/rocAL/source/loaders/video/video_read_and_decode.cpp

```diff
@@ -77,10 +77,9 @@ void VideoReadAndDecode::create(VideoReaderConfig reader_config, VideoDecoderCon
     set_video_process_count(_video_count);
     _video_decoder.resize(_video_process_count);
     _video_names = _video_prop.video_file_names;
-    _sequence_count = _batch_size / _sequence_length;
-    _decompressed_buff_ptrs.resize(_sequence_count);
-    _actual_decoded_width.resize(_sequence_count);
-    _actual_decoded_height.resize(_sequence_count);
+    _decompressed_buff_ptrs.resize(_batch_size);
```

Comment:
The variable sequence_count is not necessary now since the concept of IBS is removed in tensor , is my understanding correct ?

Reply:
Yes